tests/node/app-boot-test.js

```diff
@@ -3,19 +3,148 @@
 var path = require('path');
 var distPath = path.join(__dirname, '../../dist');
 
+QUnit.config.notrycatch = true;
```

Comment:
Yea NPM isn't that great.  The other issue is that not everyone strictly follows semver, so stupid ranged version numbers can bite you as well.  Because of this Ember CLI now adds `--save-dev --save-exact` on anything it installs.